<launch>
  <param name="robot_ip" value="10.192.1.2" />
  <node name="pointfoot_highlevel_node" pkg="pointfoot_highlevel" type="pointfoot_highlevel_node" output="screen" required="true"/>
</launch>
